### ISSUE TITLE
Recovering training hyperparameters of pre-trained models

### ISSUE THREAD

**Issue by marcozullich** [uni-medical/MedIM#8]
Hello,
Maybe I missed this information somewhere, but I think it should be useful to provide more info concerning the hyperparameters for the pre-trained models in order to allow for usability of the repository.

In particular, knowing the data preprocessing is fundamental to use correctly the models, as, e.g., bad data normalization hyperparameters may lead to completely unusable results on the same training dataset.

Additionally, could you also point to the publications/technical reports or GitHub repos for purpose of reproducing the training phase?

Thank you!

Marco
